www/assets/js/login.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+// initialize Hoodie
+var hoodie  = new Hoodie();
```

Comment:
So this is another instance of Hoodie, right? Will this override anything in `main.js`?